.github/workflows/ci-cd.yml

```
@@ -587,18 +587,214 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Run Lighthouse CI
+      - name: Install Lighthouse CI
+        run: npm install -g @lhci/cli@0.12.x
+
+      - name: Create Lighthouse configuration for deployed site
+        run: |
```

actionlint (1.7.7)

property "deploy-production" is not defined in object type {}

(expression)